packages/client/tsconfig.json

```diff
@@ -24,8 +24,9 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "preserve",
-    "composite": false
+    "jsx": "react-jsx",
+    "composite": false,
+    "noFallthroughCasesInSwitch": true
```

Comment:
@sandro768  - was there a reason for this switch-related change?  Personal preference?

Reply:
No, I haven't changed this file.

My guess is that it was done automatically by yarn upgrade.